src/ElboDeriv.jl

```diff
@@ -11,8 +11,8 @@ import ..KL
 import ForwardDiff
 import ..WCSUtils
 
-Threaded = true
-if VERSION > v"0.5.0-dev"
+Threaded = false
```

Comment:
Is the threaded stuff in ElboDeriv.jl used at all anymore?  If not, are you just leaving it in in case we change our mind in the future?  If so, could you maybe mention that in a comment?